clang/include/clang/CConv/PersistentSourceLoc.h

```diff
@@ -27,22 +27,27 @@
 
 class PersistentSourceLoc {
 protected:
-  PersistentSourceLoc(std::string f, uint32_t l, uint32_t c) : FileName(f), LineNo(l), ColNo(c),
-                                                               isValid(true) {}
+  PersistentSourceLoc(std::string f, uint32_t l, uint32_t c, uint32_t e) :
+                      FileName(f), LineNo(l), ColNo(c), ColNoE(e),
+                      isValid(true) {}
```

Comment:
Add comment to explain what the fields are for this? File, line, column-begin, and column-end, is that right? Possible to change `ColNo` to `ColNoB` (to go with `ColNoE`) ?

Reply:
Done.